src/models/schema.ts

```diff
@@ -89,7 +89,7 @@ export class Schema extends Data implements IResourceBase, ISearchable {
   }
 
   searchableText(): string {
-    return `${this.subject} ${this.version}`;
+    return this.id;
```

Comment:
This mainly ensures we don't over-match while searching for a subject. It isn't currently possible to search for a schema ID in the Topics view since we don't have any eagerly-fetched `Schema`s to attach to the subject container tree items, just the subjects themselves.